runtime/dart_isolate.cc

```diff
@@ -332,6 +334,33 @@ fml::RefPtr<fml::TaskRunner> DartIsolate::GetMessageHandlingTaskRunner() const {
   return message_handling_task_runner_;
 }
 
+bool DartIsolate::LoadLoadingUnit(intptr_t loading_unit_id,
+                                  const uint8_t* snapshot_data,
+                                  const uint8_t* snapshot_instructions) {
+  tonic::DartState::Scope scope(this);
+  Dart_Handle result = Dart_DeferredLoadComplete(loading_unit_id, snapshot_data,
+                                                 snapshot_instructions);
+  if (Dart_IsApiError(result)) {
```

Comment:
Here and elsewhere, `tonic::LogIfError` unless there is a reason this can only be an API error.